packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx

```diff
@@ -46,7 +47,9 @@ export const CoinsFilter = () => {
     const device = useSelector(selectDevice);
 
     const unavailableCapabilities = device?.unavailableCapabilities ?? {};
-    const supportedNetworks = enabledNetworks.filter(symbol => !unavailableCapabilities[symbol]);
+    const supportedNetworks = enabledNetworks
```

Comment:
This solution would fix it just for this one particular case. `enabledNetworks` is used on more places in codebase, how about storing it in redux directly in the right order?

A migration for current users would be nice.